librad/src/identities/git.rs

```diff
@@ -103,6 +103,22 @@ impl<'a, T: 'a> Identities<'a, T> {
         }
     }
 
+    /// Convenience to specialise `T` to [`VerifiedPerson`].
+    pub fn as_verified_person(&self) -> Identities<'_, VerifiedPerson> {
+        Identities {
+            repo: self.repo,
+            _marker: PhantomData,
+        }
+    }
+
+    /// Convenience to specialise `T` to [`VerifiedProject`].
+    pub fn as_verified_project(&self) -> Identities<'_, VerifiedProject> {
+        Identities {
+            repo: self.repo,
+            _marker: PhantomData,
+        }
+    }
+
```

Comment:
Nope -- casting to verified must go through verify!

Reply:
Oh wait, nevermind. I got the diff context wrong.

Reply:
:innocent: